src/SignalR/clients/ts/FunctionalTests/scripts/run-tests.ts

```diff
@@ -195,10 +195,9 @@ if (sauce) {
 }
 
 function runKarma(karmaConfig) {
-    return new Promise<karma.TestResults>((resolve, reject) => {
-        const server = new karma.Server(karmaConfig);
-        server.on("run_complete", (browsers, results) => {
-            return resolve(results);
+    return new Promise<number>((resolve, reject) => {
+        const server = new karma.Server(karmaConfig, (exitCode: number) => {
```

Comment:
I feel like I had this in the script at one point and changed it... but I think I was probably just being doofy.